src/screens/KeyManagement/OCR2KeysCreate.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import TextField from '@material-ui/core/TextField'
+import MenuItem from '@material-ui/core/MenuItem'
+import { ConfirmationDialog } from 'components/Dialogs/ConfirmationDialog'
+import { useOCR2KeyFamilyQuery } from 'hooks/queries/useOCR2KeysQuery'
+
+export interface Props {
+  showCreateKeyDialog: boolean
+  setToggleCreateKeyDialog: (toggleCreateKeyDialog: boolean) => void
+  onCreate: (chainType: string) => Promise<any>
+}
+
+export const OCR2KeysCreate: React.FC<Props> = ({
+  showCreateKeyDialog,
+  setToggleCreateKeyDialog,
+  onCreate,
+}) => {
+  const [createOCR2KeyType, setCreateOCR2KeyType] = React.useState<
+    string | undefined
+  >(undefined)
+
+  const handleChainTypeChange = (
+    event: React.ChangeEvent<{ value: unknown }>,
```

Comment:
I believe this is a html input element element, you can confirm by using devtools to see the underlying HTML element rendered by `TextField`. This will probably remove the need for you to use a type cast within the function implementation.

```suggestion
    event: React.ChangeEvent<HTMLInputElement>,
```